wpilib/wpilib/driverstation.py

```diff
@@ -254,18 +349,18 @@ def getStickPOV(self, stick: int, pov: int) -> int:
         :returns: The angle of the POV in degrees, or -1 if the POV is not
                   pressed.
         """
-        if pov < 0 or pov >= hal.kMaxJoystickPOVs:
+        if pov >= hal.kMaxJoystickPOVs:
             raise IndexError("Joystick POV is out of range")
         if stick < 0 or stick >= self.kJoystickPorts:
             raise IndexError("Joystick index is out of range, should be 0-%s" % self.kJoystickPorts)
 
         with self.cacheDataMutex:
             joystickPOVs = self.joystickPOVs[stick]
+            pov_not_available = pov >= joystickPOVs.count
 
-            if pov >= joystickPOVs.count:
-                self._reportJoystickUnpluggedWarning("Joystick POV %d on port %d not available, check if controller is plugged in\n" % (pov, stick))
-                return -1
-            return joystickPOVs.povs[pov]
+        if pov_not_available:
+            self._reportJoystickUnpluggedWarning("Joystick POV %d on port %d not available, check if controller is plugged in\n" % (pov, stick))
+        return joystickPOVs.povs[pov]
```

Comment:
This seems to no longer return `-1` in the unavailable case. Is this intentional? I don't think this is what upstream does.

Reply:
it is what upstream does